tensorflow_datasets/image_classification/horses_or_humans.py

```diff
@@ -40,7 +40,7 @@
 _IMAGE_SIZE = 300
 _IMAGE_SHAPE = (_IMAGE_SIZE, _IMAGE_SIZE, 3)
 
-_NAME_RE = re.compile(r"^(humans|horses)/[\w-]*\.png$")
+_NAME_RE = re.compile(r"^(humans|horses)(/|\\)[\w-]*\.png$")
```

Comment:
Brace means capturing group, which I don't think we want there. Will fix internally